rezervo/utils/cron_utils.py

```diff
@@ -74,37 +75,35 @@ def build_cron_job_for_class(
         *generate_booking_schedule(
             _class,
             cron_config,
-            get_integration(integration).booking_open_days_before_class,
             precheck,
         )
     )
     return j
 
 
-def generate_booking_schedule(
-    _class: Class, cron_config: Cron, booking_open_days_before: int, precheck: bool
-):
+def generate_booking_schedule(_class: Class, cron_config: Cron, precheck: bool):
```

Comment:
How about taking in `RezervoClass` instead of `Class`, and avoid adding `integration` property to `Class`?